### PR TITLE
Add class RefractivityTable. Allows fast calculation of the integrate…

### PR DESCRIPTION
**Pull request by fschlueter**
…d refractivity for straight lines in curved geometries using pre-calculated tables. Assumes a spherical earth.

The tables are anchored at the earths surface. For a given source and receiver the local zenith at the surface of the earth is calculated and along this line the refractivity is calculated. 

With the default value of 1000 zenith bins some features in the residual remain. With 10000 they vanish however the table takes quite long to initialize.

The comparison with the numerical calculation reveals some bias which is zenith angle dependent. However this is well below 0.1ns in light propagation time and thus not really import. Also within one event the introduced bias is very small.

### CODE REVIEW

File: radiotools/helper.py [hunk 1]
```diff
@@ -63,6 +65,93 @@ def datetime_to_UTC(dt):
...
+        # one intersection
+        return x0, y0
+    else:
+        # two intersections
+        d = r ** 2 - c ** 2 / r0
+        mult = np.sqrt(d / r0)
+        return x0 + b * mult, y0 - a * mult, x0 - b * mult, y0 + a * mult
+
+
+def get_zenith_angle_at_earth(zenith, station_level):
```
Comment (cg-laser) [inline radiotools/helper.py]: function documentation is missing. Also in general, the input/output parameters should be defined with

```
Parameters
------------
variablea: float
    describtion of the variable

File: radiotools/helper.py [hunk 2]
```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import, division, print_function  # , unicode_literals
+from radiotools.atmosphere import models as atm
```
Comment (cg-laser) [inline radiotools/helper.py:4]: does this import trigger the calculation of the tables? If yes, please move it to the function where you need it.

**Review by cg-laser — CHANGES_REQUESTED** [review]
I didn't check if the code works but I have a few general comments. 

Can you add a documentation to all the functions/class? (I know I have been sloppy in the past, but for new function we should try to make a better job) . 

Do you want to add a unit test?

File: radiotools/helper.py (hunk 2 above)
Comment (fschlueter) [reply radiotools/helper.py:4]: No it should not.

File: radiotools/helper.py (hunk 1 above)
Comment (fschlueter) [reply radiotools/helper.py]: I agree this is missing. I will add this during the next week.

**Comment by fschlueter** [comment]
Yes I can add a unit test.

File: radiotools/atmosphere/refractivity.py [hunk 3]
```diff
@@ -6,80 +6,186 @@
...
+    refractivity : float
+        N(h)
+    """
+
     a = 325e-6
     b = 0.1218e-3
     return 1 + a * np.exp(-b * h)
 
 
+def get_refractivity_between_to_points_numerical(p1, p2, atm_model, refractivity_at_sea_level, debug=False):
```
Comment (cg-laser) [inline radiotools/atmosphere/refractivity.py]: typo in function name "to" -> "two"

**Review by cg-laser — APPROVED** [review]
looks good except for the typo in the function name.

**Pull request merged**